src/components/Slider/Slider.jsx

```diff
@@ -0,0 +1,242 @@
+   *  - check Dev Tools for found available Custom/Consumer Classes
+   */
+  classNameBase: PropTypes.string,
+  /**
+   * Color Mode (primary/positive/negative) of the component (Slider)
+   */
+  color: PropTypes.oneOf(Object.values(Slider.colors)),
+  /**
+   * Unique TestId - can be used as Selector for integration tests and other needs (tracking, etc)
+   */
+  "data-testid": PropTypes.string,
+  /**
+   * If set to true, Component (Slider) will be disabled
+   *  - impossible to change value of component (Slider)
+   *  - visual changes (opacity)
+   */
+  disabled: PropTypes.bool,
+  /**
+   * Attribute `id` to be added to the component-root-node
+   */
+  id: PropTypes.string,
+  /**
+   * If true switch slider to RRange mode (two Thumbs)
+   */
+  isRange: PropTypes.bool,
+  /**
+   * Max range value of the component (Slider)
+   */
+  max: PropTypes.number,
+  /**
+   * Min range value of the component (Slider)
+   */
+  min: PropTypes.number,
+  /**
+   * Optional onChange callback (for outer trigger or Controlled mode)
+   * - required in Controlled Mode
+   */
+  onChange: PropTypes.func,
+  /**
+   * Step for Component (Slider) changes
+   */
+  step: PropTypes.number,
+  /**
+   * Always show `value` instead of Tooltip
+   */
+  showValue: PropTypes.bool,
+  /**
+   * Size small/medium/large of the component (Slider)
+   */
+  size: PropTypes.oneOf(Object.values(Slider.sizes)),
+  /**
+   * Current/selected value of the range of the Component (Slider)
+   *   - should be used in Controlled Mode only
+   *   - in isRange mode should be an array of two numbers
+   */
+  value: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.number), PropTypes.number]),
+  /**
+   * Default `value` if value not specified
+   *  - should be used in NON-Controlled mode to set initial Value
+   *  - in isRange mode should be an array of two numbers
+   */
+  valueDefault: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.number), PropTypes.number]),
+  /**
+   * Formatter function `value => formattedValue`
+   * default formatter return `${value}%`
+   */
+  valueFormatter: PropTypes.func,
+  /**
+   * Text/presentation of current/selected value
+   *  - should be used in Controlled Mode only
+   */
+  valueText: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.string), PropTypes.string]),
+
+  // ------ Additional subcomponents' props
+  /**
+   * Show selected from Slider range value
+   */
+  indicateSelection: PropTypes.bool,
+  /**
+   * Options for initial/start/prefix element, it can be one of:
+   *  - Any Component (react component, node, text, number etc.)
+   *  - Or it can be an object of options for Icons component (see Icon components props)
+   *  - Or it can be an object for Label (Icon, Title - and other components)
+   *  - Or it can be Render Props Function witch are getting value and valueText
+   */
+  prefix: PropTypes.oneOfType([
+    PropTypes.shape({
+      icon: PropTypes.string.isRequired
```

Comment:
Is it the our design system icon? If so it can be also a function

Reply:
Done!